opentelemetry-jaeger/src/lib.rs

```diff
@@ -447,7 +444,9 @@ impl PipelineBuilder {
     /// Initialize a new exporter.
     ///
     /// This is useful if you are manually constructing a pipeline.
-    pub fn init_exporter(self) -> Result<Exporter, Box<dyn Error + Send + Sync + 'static>> {
+    pub fn init_exporter(
+        self,
+    ) -> Result<Exporter, Box<dyn std::error::Error + Send + Sync + 'static>> {
```

Comment:
sorry can probably update this error as well

Reply:
hard to catch them all when reviewing 😬

Reply:
😝 Sorry I should have caught those. It's just too many std errors there. Hard to find out all of them. I _think_ I get them all in zillion and jaeger this time. Let me know if I missed some